docs/architecture/adr-001.md

```diff
@@ -23,7 +26,17 @@ In addition to that, the "x/cronos" module contains Gravity Bridge-related code
 
 Existing integration tests related to Gravity Bridge can be temporarily disabled and later enabled when Gravity Bridge is added back.
 
-Once the Gravity Bridge code is added back, the x/cronos module `ConsensusVersion` should be increased and the corresponding upgrade handler should be added.
+Once the Gravity Bridge code is added back (after mainnet launch), the x/cronos module `ConsensusVersion` should be increased and the corresponding upgrade handler should be added.
```

Comment:
should those (module version + upgrade handler) be conditionally added/changed as well?

Reply:
I think we can avoid this for now. 

We could keep the same consensus version for both binary (with and without gravity) and increase it when we decide to release gravity to mainnet (at that time, we can also remove the conditional compilation)

Reply:
ok, then it's good to move or rephrase this sentence

Reply:
sure, changed